webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -104,6 +107,18 @@ class BreadcrumbBar extends React.Component {
         {!isTitle && <hr className="breadcrumb-line" />}
       </div>
     );
+
+    if (wrapWithReactRouter) {
+      return (
+        <Router history={history}>
```

Comment:
How about using `withRouter` HOC? https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/withRouter.md

Reply:
That would not work here: Uncaught Invariant Violation: You should not use <Route> or withRouter() outside a <Router>